plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/command/RefreshResourceHandler.java

```diff
@@ -18,20 +20,28 @@
 import org.eclipse.core.runtime.Status;
 import org.eclipse.core.runtime.jobs.Job;
 import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.ui.IWorkbenchPart;
 import org.eclipse.ui.handlers.HandlerUtil;
 import org.jboss.tools.openshift.common.core.connection.IConnection;
 import org.jboss.tools.openshift.internal.common.core.job.AbstractDelegatingMonitorJob;
 import org.jboss.tools.openshift.internal.common.core.job.JobChainBuilder;
+import org.jboss.tools.openshift.internal.common.ui.OpenShiftCommonUIActivator;
 import org.jboss.tools.openshift.internal.common.ui.utils.UIUtils;
-import org.jboss.tools.openshift.internal.ui.OpenShiftUIActivator;
+//import org.jboss.tools.openshift.internal.ui.OpenShiftUIActivator;
+
+
+
+
+
 
 import com.openshift.client.OpenShiftException;
-import com.openshift.client.Refreshable;
+import com.openshift.client.IRefreshable;
 
```

Comment:
@adietish fixing now.  I renamed it for consistency, but i dont see it pushed to the server.  Hoping I didnt miss something else i changed